insights/specs/insights_archive.py

```diff
@@ -182,6 +182,7 @@ class InsightsArchiveSpecs(Specs):
     ovs_vsctl_list_bridge = simple_file("insights_commands/ovs-vsctl_list_bridge")
     ovs_vsctl_show = simple_file("insights_commands/ovs-vsctl_show")
     package_provides_command = glob_file("insights_commands/echo_*java*")
+    partitions = simple_file("/proc/partitions")
```

Comment:
It's no need to add `simple_file` to the insights_archive.py, if it's a `simple_file` originally.

Reply:
Removed partitions from insights_archive.py file.